CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
    + Note: data representations require Postgres 10 (Postgres 11 if using `IN` predicates); data representations are not implemented for RPC
  - #2647, Allow to verify the PostgREST version in SQL: `select distinct application_name from pg_stat_activity`. - @laurenceisla
  - #2856, Add the `--version` CLI option that prints the version information - @laurenceisla
+ - #1655, Add the number of rows affected to the `hint` field of response of `SingularityError`. - @taimoorzaeem
```

Comment:
```suggestion
 - #1655, Add the number of rows affected to the `hint` field of a singular response error. - @taimoorzaeem
```

`SingularityError` may confuse users, because it's an internal identifier in the code.